docs/UserGuide.md

```diff
@@ -156,71 +164,288 @@ To view all our features, you may visit out [features section](#features).
 
 ### Adding an assignment
 
+```put command first```
+
+**Purpose:**
+
+<div markdown="block" class="alert alert-danger">
+
+**ALERT**
+Put alerts here, if any
+</div>
+
+<div markdown="block" class="alert alert-info">
+
+**Note**
+Put notes here
+</div>
+
+**Examples:** show some examples with the parameters
+
+**When you might use it:** give some use cases
+
 [Scroll back to Table of Contents](#table-of-contents)
 
 ### Editing an assignment
 
+```put command first```
+
+**Purpose:**
+
+<div markdown="block" class="alert alert-danger">
+
+**ALERT**
+Put alerts here, if any
+</div>
+
+<div markdown="block" class="alert alert-info">
+
+**Note**
+Put notes here
+</div>
+
+**Examples:** show some examples with the parameters
+
+**When you might use it:** give some use cases
+
 [Scroll back to Table of Contents](#table-of-contents)
 
 ### Deleting an assignment
 
+```delete-a INDEX```
+
+**Purpose:** To delete the specified doctor from CampusCompanion
+
+<div markdown="block" class="alert alert-danger">
+
+**ALERT**
+This command cannot be undone. If you remove the wrong assignment, you will have to add it back using ``add-a``.
+
+</div>
+
+
+<div markdown="block" class="alert alert-info">
+
+**Note**
+
+- The index refers to the index number in the displayed assignment list. If you want to find out the index of the 
+assignment you want to delete, you can use ``list-a`` to view all the current assignment you have.
+
+- The first assignment in the list may not be index 1. Please refer to [parameter summary](#parameter-summary) 
+if you want to understand how to find the index number.
+
+</div>
+
+**Examples:**
+- ``list-a`` to list all the assignments, followed by ``delete-a 1`` to delete assignment with **index** 1.
+- ``find-a CS2103T`` to find assignments which have "CS2103T" in the name, followed by ``delete-a 1`` to delete assignment with **index** 1.
```

Comment:
A bit confusing, may give the perception that it'll delete the 1st assignment from the return of `find-a CS2103T`, would be good to specify that the thing you want to delete is at index 1.

I think that another way to do this would be adding "if the assignment with the keyword CS2103T that you want to delete is at index 1, run `delete-a 1`"